azure-pipelines.yml

```diff
@@ -28,13 +28,20 @@ variables:
     value: docker-python-databricks
   - name: imageRepository_snowpark
     value: docker-python-snowpark
+  - name: changedProjects_python38
+    value: 0
+  - name: changedProjects_python310
+    value: 0
+  - name: changedProjects_pythonSnowpark
+    value: 0
```

Comment:
I don't like this much. It's yet another place where we have to manually update when new "project" is added. Is it really nessecary?

Reply:
Ok, I put the set into the script instead